### PR TITLE
Include triggers, pileup jet id, some additional lepID bits (<= still more needed) 

### PR DESCRIPTION
**Pull request by jsalfeld**

### CODE REVIEW

File: Nero/src/NeroLeptons.cpp [hunk 1]
```diff
@@ -13,7 +13,7 @@ NeroLeptons::NeroLeptons():
 
     mMinPt_mu = 10;
     mMinEta_mu = 2.4;
-    mMaxIso_mu = 0.2;
+    mMaxIso_mu = 0.4;
```
Comment (amarini) [inline Nero/src/NeroLeptons.cpp]: this value is actually overwritten here (keep the -1 that is less strict)
https://github.com/MiT-HEP/NeroProducer/blob/master/Nero/python/Nero_cfi.py#L155

File: Nero/src/NeroLeptons.cpp [hunk 2]
```diff
@@ -67,18 +67,31 @@ int NeroLeptons::analyze(const edm::Event & iEvent)
         float puiso = mu.pfIsolationR04().sumPUPt;
         float totiso = chiso + TMath::Max( niso + phoiso - .5*puiso, 0. ) ;
     
-        if ( mMaxIso_mu > 0 and totiso/mu.pt() > mMaxIso_mu ) continue;
+		bool isFake = 0;
 
+        if ( mMaxIso_mu > 0 and totiso/mu.pt() > mMaxIso_mu ) continue;
+		if ( totiso/mu.pt() < 0.4){ isFake = 1; }
+
```
Comment (amarini) [inline Nero/src/NeroLeptons.cpp]: Can you confirm is not the opposite ? (totIso > XXX) the terminology fake seems misleading

**Comment by amarini** [comment]
You probably want to port some of this stuff to the master (80X, eg the LepFake)  but not all (eg the pileupId).

**Comment by jsalfeld** [comment]
yes, bit definition will need one more work through

**Pull request merged**